regression/cbmc-java/stack_var1/stack_var1.j

```diff
@@ -0,0 +1,22 @@
+.class public stack_var1
```

Comment:
these are the source files I used to create the classes. This is very bytecode specific, in particular the `dup2_x?` are very rare, so I used the jasmin assembler to create the regression tests.